src/Midgard/CreatePHP/Mapper/BaseDoctrineRdfMapper.php

```diff
@@ -58,9 +61,10 @@ public function store(EntityInterface $entity)
      *
      * use getRealClass if className names a doctrine proxy class.
      */
-    public function canonicalName($className)
+    public function objectToName($object)
     {
-        $refl = new \ReflectionClass($className);
+        $refl = new \ReflectionClass($object);
+        $className = $refl->getName();
```

Comment:
as we have the object now, we should not use reflection but instead simply do this (no need even for an "if")

    \Doctrine\Common\Util\ClassUtils::getClass($object);

(and lets add a use statement for Doctrine\Common\Util\ClassUtils)